src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -860,32 +935,31 @@ collate_opt:
     $$ = string($2)
   }
 
-column_key_opt:
-  {
-    $$ = ColKeyNone
-  }
-| PRIMARY KEY
+column_primary_key_opt:
+PRIMARY KEY
   {
     $$ = ColKeyPrimary
   }
 | KEY
   {
-    $$ = ColKey
+    // KEY is normally a synonym for INDEX. The key attribute PRIMARY KEY
+    // can also be specified as just KEY when given in a column definition.
+    // See http://dev.mysql.com/doc/refman/5.7/en/create-table.html
+    $$ = ColKeyPrimary
```

Comment:
Key != PRIMARY KEY
ColKey != ColKeyPrimary, 
you should review it again

Reply:
> Key != PRIMARY KEY
> ColKey != ColKeyPrimary,
> you should review it again

In column definition, key is specified as primary key, the comment has given the doc next, maybe you can confirm again: http://dev.mysql.com/doc/refman/5.7/en/create-table.html
```
column_definition:
    data_type [NOT NULL | NULL] [DEFAULT default_value]
      [AUTO_INCREMENT] [UNIQUE [KEY]] [[PRIMARY] KEY]
      [COMMENT 'string']
      [COLLATE collation_name]
      [COLUMN_FORMAT {FIXED|DYNAMIC|DEFAULT}]
      [STORAGE {DISK|MEMORY}]
      [reference_definition]
```